remote_vector_index_builder/core/common/models/index_builder/index_hnsw_cagra_config.py

```diff
@@ -0,0 +1,28 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+from dataclasses import dataclass
+
+
+@dataclass
+class IndexHNSWCagraConfig:
+    # expansion factor at search time
+    ef_search: int = 256
+
+    # expansion factor at construction time
+    ef_construction: int = 40
```

Comment:
Please use 100, 100 here. As those are the defaults of k-NN

Reply:
updated